sharktank/sharktank/layers/paged_attention.py

```diff
@@ -945,6 +958,8 @@ def repeat_kv(x: torch.Tensor) -> torch.Tensor:
                 scale=None,  # defaults to 1/sqrt(dim)
             )
 
+        return attn_output
```

Comment:
Is there a reason we changed to this formatting? Generally returning from each `if ` makes readability a little easier as you can identify early termination (though if we do this the `else` above should just be remove and `scaled_dot_product_attention` should just happen as a follow up to each other else.